charts/rd-user-profile-api/values.yaml

```diff
@@ -11,6 +11,10 @@ java:
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     OIDC_ISSUER_URL: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
     OPEN_ID_API_BASE_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
+    IDAM_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
+    AUTH_IDAM_CLIENT_BASEURL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
+    CCD_URL: http://ccd-user-profile-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
CCD_URL is not required can you please delete

Reply:
Addressed here: https://github.com/hmcts/rd-user-profile-api/pull/420/commits/8d735e656e01bf32b404cc69d4b8fd1e38b955dc